javascript/ql/src/experimental/Security/CWE-347-noVerification/jwtSimple.ql

```diff
@@ -0,0 +1,53 @@
+/**
+ * @name JWT missing secret or public key verification
```

Comment:
Again, it would be better to combine this with your existing `JsonWebToken.ql` query, instead of having one query pr. library.